packages/Disclosure/src/disclosure.scss

```diff
@@ -0,0 +1,107 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-disclosure-border-radius: #{tokens.$border-radius-sm};
+  --ids-disclosure-color: #{tokens.$grey-800};
+  --ids-disclosure-font-family: #{tokens.$primary-font-family};
+  --ids-disclosure-font-size: #{tokens.$font-size-3};
+  --ids-disclosure-shadow: #{tokens.$shadow-6};
+
+  /* Header/Button */
+  --ids-disclosure-header-background: #{tokens.$samoyed};
+  --ids-disclosure-header-line-height: #{tokens.$line-height-xxl};
+  --ids-disclosure-header-padding: #{tokens.$space-4};
+  --ids-disclosure-header-icon-margin: 0 #{tokens.$space-4} 0 0;
+  --ids-disclosure-header-title-font-family: #{tokens.$secondary-font-family};
+  --ids-disclosure-header-title-font-size: #{tokens.$font-size-6};
+  --ids-disclosure-header-title-font-weight: #{tokens.$font-weight-medium};
+  --ids-disclosure-header-title-line-height: #{tokens.$line-height-lg};
+  --ids-disclosure-header-desc-color: #{tokens.$grey-600};
+  --ids-disclosure-header-content-margin: calc(-#{tokens.$space-1} / 2) -#{tokens.$space-2};
+  --ids-disclosure-header-content-padding: calc(#{tokens.$space-1} / 2) #{tokens.$space-2};
+  --ids-disclosure-header-chevron-color: #{tokens.$grey-600};
+  --ids-disclosure-header-chevron-margin: 0 0 0 #{tokens.$space-2};
+
+  /* Content */
+  --ids-disclosure-content-border: 1px solid #{tokens.$grey-200};
+}
+
+.ids-disclosure {
+  box-shadow: var(--ids-disclosure-shadow);
+  font-family: var(--ids-disclosure-font-family);
+  font-size: var(--ids-disclosure-font-size);
+
+  .ids-disclosure__header {
+    align-items: center;
+    background: var(--ids-disclosure-header-background);
+    border: none;
+    border-radius: var(--ids-disclosure-border-radius);
```

Comment:
once opened there should be no border radius at the bottom of the button.

<img width="160" alt="Capture d’écran, le 2023-02-21 à 16 46 29" src="https://user-images.githubusercontent.com/2379339/220465865-3d1f7252-6722-409d-b7b8-e99fed76ca65.png">

Reply:
ya that was supposed to be on the ids-disclosure element. Good catch!